packages/reverse-exports/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "reverse-exports",
```

Comment:
@ef4 how do you feel about this not being `@embroider/reverse-exports`? I figured it would be nice to just have it as `reverse-exports` in case anyone else needed it or in case we wanted to extract the package from the monorepo at any point

Reply:
@lolmaus let's reset this back to `@embroider/reverse-exports` and updated where it needs to be 👍

Reply:
@mansona Done.